src/PeopleSDKAdapter.test.js

```diff
@@ -0,0 +1,115 @@
+import {isObservable} from 'rxjs';
+
+import PeopleSDKAdapter, {USER_PRESENCE_UPDATE_EVENT} from './PeopleSDKAdapter';
+import createMockSDK from './__mocks__/sdk';
+
+describe('People SDK Adapter', () => {
+  let peopleSDKAdapter;
+  let mockSDK, mockSDKPerson;
+
+  beforeEach(() => {
+    mockSDK = createMockSDK();
+    peopleSDKAdapter = new PeopleSDKAdapter(mockSDK);
+
+    mockSDKPerson = {
+      id: 'id',
+      emails: ['email@cisco.com'],
+      displayName: 'Webex Components',
+      nickName: 'Teams',
+      firstName: 'Webex',
+      lastName: 'Components',
+      avatar: 'avatar',
+      orgId: 'orgID',
+      created: 'created',
+      type: 'person',
+      status: 'active',
+    };
+  });
+
+  describe('getPerson() functionality', () => {
+    test('returns an observable', () => {
+      expect(isObservable(peopleSDKAdapter.getPerson())).toBeTruthy();
+    });
+
+    test('returns a person in a proper shape', (done) => {
+      mockSDK.people.get = jest.fn(() => Promise.resolve(mockSDKPerson));
+      mockSDK.internal.presence.list = jest.fn(() => Promise.resolve({statusList: [{status: 'active'}]}));
+
+      peopleSDKAdapter.getPerson('id').subscribe((person) => {
+        expect(person).toEqual(
```

Comment:
@lalli-flores discovered that [toMatchObject](https://jestjs.io/docs/en/expect#tomatchobjectobject) cleans up the expectation block a bit. We should standardize on that.